examples/sveltekit/example.env

```diff
@@ -0,0 +1,3 @@
+# Update these with your Supabase details from your project settings > API
```

Comment:
nit: I'd call this `.env.example` and maybe add instructions to readme to copy it into `.env` file?